Lib/argparse.py

```diff
@@ -703,13 +703,10 @@ class ArgumentDefaultsHelpFormatter(HelpFormatter):
     """
 
     def _get_help_string(self, action):
-        """
-        Add the default value to the option help message.
+        """Add the default value to the option help message.
```

Comment:
I would convert the docstring into comment or remove it. No other private method in this class hierarhy has a docstring. And the text just repeats what the code do, but less accurately.

Reply:
@serhiy-storchaka That seems like a good idea. I was focused on the _coherence_ of the text (or lack thereof), but looking at it again I agree it doesn't really add much even when it makes sense.

Reply:
I decided to just yank it entirely, fully in agreement with:

> the text just repeats what the code do, but less accurately.